src/Rector/AbstractRector.php

```diff
@@ -320,6 +323,10 @@ private function postRefactorProcess(
             return $originalNode;
         }
 
+        if ($refactoredNode instanceof Expr && $originalNode instanceof Stmt) {
+            $refactoredNode = new Expression($refactoredNode);
+        }
```

Comment:
Let's see if this wrapper resolve it.

Reply:
This should be avoided, as it allows to create weak rules.

Reply:
Ok, I will revert, currently looking on why it still error in phpstan 1.10.3

https://github.com/rectorphp/rector-src/actions/runs/5966007434/job/16184720897#step:14:64

Reply:
Reverted